testsuite/MDAnalysisTests/analysis/test_hydrogenbonds_analysis.py

```diff
@@ -552,3 +552,28 @@ def test_count_by_type(self, h):
 
         counts = h.count_by_type()
         assert int(counts[0, 2]) == ref_count
+
+
+class TestHydrogenBondAnalysisEmptySelections:
+
+    @pytest.fixture()
+    def h(self):
+
+        universe = MDAnalysis.Universe(waterPSF, waterDCD)
+        kwargs = {
+            'donors_sel': '',
+            'hydrogens_sel': '',
+            'acceptors_sel': '',
+        }
+
+        h = HydrogenBondAnalysis(universe, **kwargs)
+        h.run()
+
+        return h
```

Comment:
Minor point--is it likely this fixture will be used again? Just wondering about the value of the abstraction/fixture magic, but probably doesn't matter much.

Reply:
Yeah good point, I can't see it being used gain so I've removed the fixture